src/main/kotlin/graphics/scenery/volumes/VolumeManager.kt

```diff
@@ -429,6 +434,8 @@ class VolumeManager(
                     currentProg.setConverter(i, state.converterSetup)
                     currentProg.registerCustomSampler(i, "transferFunction", state.transferFunction)
                     currentProg.registerCustomSampler(i, "colorMap", state.colorMap)
+                    currentProg.setCustomFloatArrayUniformForVolume(i,"slicingPlanes",4,state.node.slicingArray())
```

Comment:
Spaces between arguments missing (I know, I know, we need a code style file :-D)